modules/webpack.config.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * webpack.config.js - webpack configuration script for testing ilib-localeinfo
+ *
+ * Copyright © 2022, JEDLSoft
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var webpack = require('webpack');
+var path = require('path');
+
+module.exports = {
+    mode: "development",
+    entry: "./node_modules/ilib-localeinfo",
+    output: {
+        path: path.resolve(__dirname, 'test'),
+        filename: "moduleTest.js",
+        publicPath: "test/",
+        library: {
+            name: "LocaleInfoModule",
+            type: "var"
+        }
+    },
+    externals: {
+        "./NodeLoader.js": "NodeLoader",
+        "./QtLoader.js": "QtLoader",
+        "./RhinoLoader.js": "RhinoLoader",
+        "./NashornLoader.js": "NashornLoader",
+        "./RingoLoader.js": "RingoLoader",
+        "log4js": "log4js",
+        "nodeunit": "nodeunit"
```

Comment:
I think log4js and nodeunit are not needed. no?

Reply:
log4js is needed yes, but nodeunit is not. These lines specify that those identifiers are provided by some code outside of the webpacked bundle, so they do not need to be included in the webpack bundle. The libraries ilib-localeinfo and the ones it depends on use log4js if it is there. If not, it will stay quiet. But, because it is referenced (but not used!), webpack will nonetheless still attempt to include the code for it. That is, unless you use the line above to tell it not to bother trying to include it. I'll remove the nodeunit part.